packages/terra-list/docs/guides/MultiSelectList.md

```diff
@@ -25,14 +25,14 @@ As a precaution we can prevent default on the event, in case the list has an anc
     event.preventDefault();
   }
 ```
-Terra list comes with additional helpers to manage state, in this case we want to determine if the selection has selected or unselected the list item key in our state. So we use the utility method "updatedMulitSelectedKeys", which returns an array of the keys following the addition or removing of the key passed. We then set this as our state.
+Terra list comes with additional helpers to manage state, in this case we want to determine if the selection has selected or unselected the list item key in our state. So we use the utility method `updatedMulitSelectedKeys`, which returns an array of the keys following the addition or removing of the key passed. We then set this as our state.
```

Comment:
I think the middle of this linke should be `manage state. In this case,`

Reply:
Updated.